readme.txt

```diff
@@ -190,6 +190,9 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= TBD =
```

Comment:
Instead of using the acronym, write out the full name.

Reply:
Done.